src/Broker/Broker.php

```diff
@@ -56,10 +61,13 @@ public function __construct(
 		array $methodsClassReflectionExtensions,
 		array $dynamicMethodReturnTypeExtensions,
 		array $dynamicStaticMethodReturnTypeExtensions,
+		array $dynamicFunctionReturnTypeExtensions,
 		FunctionReflectionFactory $functionReflectionFactory,
 		FileTypeMapper $fileTypeMapper
 	)
 	{
+		$dynamicFunctionReturnTypeExtensions[] = new PhpCoreFunctionsReturnTypeExtension();
```

Comment:
Config neon is absolutely better place, but how can I ensure last position? This solution seems safer to me.

Tests depends on this extension too.

Reply:
I think it can go first similarly to PhpClassReflectionExtension. But you need to update the isFunctionSupported return value. Register in the DIC and handle it in th BrokerFactory.